src/examples/mailchimp/src/backend/schema/category/data-entity.ts

```diff
@@ -0,0 +1,14 @@
+import { BaseDataEntity } from '@exogee/graphweaver';
+
+export class CategoryDataEntity implements BaseDataEntity {
```

Comment:
I think of entities in `/schema` as being graphql entities, and these look like base data entities. Is this how AIA does it?

Reply:
Fixed. They are restructured to entities/category.ts and schema/category